db/migrate/20171106143927_downcase_resource_group_ems_ref_for_azure.rb

```diff
@@ -0,0 +1,11 @@
+class DowncaseResourceGroupEmsRefForAzure < ActiveRecord::Migration[5.0]
+  class ResourceGroup < ActiveRecord::Base; end
```

Comment:
Is this an STI class?

Reply:
There is a `type` column. Should I have subclassed that in the provider repo? If so I can add it in the other PR and update that column as well as part of the data migration as well.

Reply:
If there is a type column, then it's STI.  You must

- create a record in your test with the type column set to a "real" model, to ensure it works
- when that test fails, you will need to set the inheritance column like we do in other model.

> Should I have subclassed that in the provider repo?

They should already be subclasses in the other repo, however migrations must work by not having those models present (which is why we use a dummy Rails app here).

Reply:
The subclass was recently added to the Azure provider repo.